forge.config.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
     packagerConfig: {
-        icon: './src/assets/icon'
+        icon: './src/assets/icon',
```

Comment:
Remember to check building for both Windows and Linux. I recall adding the name field in the packagerConfig to fix one of the platforms, but then the other one didn't work. Things I noticed were the application name and menu items depended on this.

Reply:
I had similar issue earlier where if the name in package.json does not match the name in forge.config.js, I would get build errors. Having them similar produced successful builds:
![image](https://user-images.githubusercontent.com/8902531/184993359-83021bee-4314-48c6-8173-a5166b78e7b9.png)